### PR TITLE
[Fix] GOG stuck progress on download + Add 7 new languages

### PR DESCRIPTION
**Pull request by flavioislima**
- Updated GOGDL with the fix;
- Added new languages that were +95% translated on Weblate
- Backported #1797 

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,18 +89,21 @@ Heroic is built with Web Technologies like: TypeScript, React, NodeJS and Electr
 - French
 - German
 - Greek
+- Japanese
```
Comment (flavioislima) [inline README.md:92]: Japanese was translated but was missing on readme. While Romanian was here but it was not available to select.

File: src/screens/Settings/index.tsx [hunk 2]
```diff
@@ -343,6 +343,7 @@ function Settings() {
       defaultWinePrefix,
       disableController,
       discordRPC,
+      downloadNoHttps,
```
Comment (flavioislima) [inline src/screens/Settings/index.tsx:346]: this was a new fix in beta that I am adding here as well.

File: electron/gog/games.ts [hunk 3]
```diff
@@ -132,8 +132,8 @@ class GOGGame extends Game {
     const etaMatch = data.match(/ETA: (\d\d:\d\d:\d\d)/m)
     const bytesMatch = data.match(/Downloaded: (\S+) MiB/m)
     const progressMatch = data.match(/Progress: (\d+\.\d+) /m)
-    if (etaMatch && bytesMatch && progressMatch) {
-      const eta = etaMatch[1]
+    if (bytesMatch && progressMatch) {
```
Comment (flavioislima) [inline electron/gog/games.ts:135]: Removed the eta dependency just in case someone has a pretty slow connection to download a huge game.

**Review by imLinguin — APPROVED** [review]
Looks good to me

**Pull request merged**